cassandra-driver.gemspec

```
@@ -31,7 +31,8 @@ Gem::Specification.new do |s|
   end
 
   s.add_runtime_dependency 'ione', '~> 1.2'
+  s.add_runtime_dependency 'sorted_set', '~> 1.0'
 
-  s.add_development_dependency 'bundler', '~> 1.6'
+  s.add_development_dependency 'bundler', '~> 2.4'
```

```suggestion
  s.add_development_dependency 'bundler'
```
versionless so the version can float if necessary for the current ruby version